crategen/converters/abstract_converter.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+class AbstractConverter(ABC):
+
+    @abstractmethod
+    def convert_to_wrroc(self, data):
+        """Convert data to WRROC format"""
+        pass
```

Comment:
You don't need `pass` if your function signature contains a docstring

Reply:
I will remove the `pass` statement from the abstract methods in the AbstractConverter class.